qe-tests/README.md

```diff
@@ -1 +1,7 @@
-# This folder is for writting the tests comming from QE
+# End-to-end Cypress tests for tackle-ui
+
+##### Get Started ####
+# Git clone https://github.com/carlosthe19916/tackle-ui-testing-poc.git
+# Cd qe-tests
+# npm install .
+
```

Comment:
Can we add steps to open cypress runner (npx cypress open) or run them headless (npx cypress run)